src/component/Carousel/Carousel.js

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { Carousel } from "react-bootstrap";
+import { Link } from "react-router-dom";
+import "./Carousel.css";
+
+export function Carouselbar() {
+  return (
+    <div className="home" id="home">
+    <div className="slide">
+    <Carousel fade>
+      <Carousel.Item className="image">
+        <img
+          className="travel"
+          src={require("../../assets/images/trekking.png")}
```

Comment:
you can put all your images in the public folder can give the path directly here. no need to use require.